torch/_inductor/codecache.py

```diff
@@ -2948,12 +2948,20 @@ def _codegen_glue(cls, argtypes, headerfile):
     @classmethod
     @functools.lru_cache(None)
     def config_hash(cls):
+        from torch._inductor.cpp_builder import CppBuilder, CppOptions
+
+        dummy_builder = CppBuilder(
+            name="O",
+            sources="I",
+            BuildOption=CppOptions(compile_only=False),
+        )
+        command_line = dummy_builder.get_command_line()
         return sha256_hash(
             "\n".join(
                 [
                     cls.glue_template,
                     f"{cls.cpu_cache_size()}",
-                    cpp_compile_command("I", "O"),
+                    command_line,
```

Comment:
just wondering, would it be simpler to move the `CppBuilder` thing into the implementation of `cpp_compile_command`?

Reply:
@jgong5 current design, `CppBuilder` and `CppOptions` are splited, but maybe we can optimize and combine them together. Let's land code firstly.

Reply:
I mean we can still keep the `cpp_compile_command` implemented with the `CppBuilder` stuff. `cpp_compile_command` is also used in other places?

Reply:
![image](https://github.com/pytorch/pytorch/assets/8433590/dc464868-99a3-448d-aca5-6b0ee9fd0487)

Hi @jgong5 current design is that, we can choose many types of build options on demand. It is also easy for derive more new devices.
one function seems not good to integrate into current design.